scripts/qa_crowbarsetup.sh

```diff
@@ -3002,6 +3061,44 @@ function oncontroller_tempest_legacy()
     return $tempestret
 }
 
+function oncontroller_manila_generic_driver_setup()
+{
+    local service_image_url=http://clouddata.cloud.suse.de/images/other/manila-service-image.qcow2
+    local service_image_name=manila-service-image.qcow2
+    local sec_group="manila-service"
+    local neutron_net=$sec_group
+
+    wget --progress=dot:mega -nc -O $service_image_name \
+        "$service_image_url" || complain 73 "manila image not found"
+
+    . .openrc
+    openstack image create --file $service_image_name --public \
+        --disk-format qcow2 manila-service-image
+    nova flavor-create manila-service-image-flavor 100 256 0 1
+
+    nova secgroup-create $sec_group "$sec_group description"
+    nova secgroup-add-rule $sec_group icmp -1 -1 0.0.0.0/0
+    nova secgroup-add-rule $sec_group tcp 22 22 0.0.0.0/0
+    nova secgroup-add-rule $sec_group tcp 2049 2049 0.0.0.0/0
+    nova secgroup-add-rule $sec_group udp 2049 2049 0.0.0.0/0
+    nova secgroup-add-rule $sec_group udp 445 445 0.0.0.0/0
+    nova secgroup-add-rule $sec_group tcp 445 445 0.0.0.0/0
+    nova secgroup-add-rule $sec_group tcp 137 139 0.0.0.0/0
+    nova secgroup-add-rule $sec_group udp 137 139 0.0.0.0/0
```

Comment:
How about documenting it in form of a script that we ship with manila?
Then we could just call it with a $sec_group value
and maybe it could do even more of the tricky setup parts.